package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" ",
-    "build": "cd website && npm run build"
+    "copy": "copyfiles -u 1 src/**/**/* src/**/* src/* website",
+    "build": "cd website&& npm run copy && workbox injectManifest workbox-config.js && npm run build",
```

Comment:
Why are you running build script inside build script? Wouldn't it create a circular dependency?